src/pyDataverse/docs/source/user/basic-usage.rst

```diff
@@ -72,10 +72,20 @@ can then be used (e. g. :meth:`json() <requests.Response.json>`).
 Create Dataverse Collection
 -----------------------------
 
-The top-level data-type in the Dataverse software is called a Dataverse collection, so we will
-start with that.
+The top-level data-type in the Dataverse software is called a Dataverse collection, so we will start with that. 
+Take a look at the figure below to better understand the relationship between a Dataverse collection, a dataset, and a datafile. 
 
-First, instantiate a :class:`Dataverse <pyDataverse.models.Dataverse>`
+.. figure:: ../_images/collection_dataset.png
+   :align: center
+   :alt: collection dataset datafile
+
+A dataverse collection (also known as a :class:`Dataverse <pyDataverse.models.Dataverse>`) acts as a container for your :class:`Datasets<pyDataverse.models.Dataverse>`. 
```

Comment:
It is a great idea for beginners to share a bit of the context before getting started.  Such as what the data-structure is.